TheMovieDatabase/ViewControllers/Feed/View/FeedViewController.swift

```diff
@@ -45,6 +46,14 @@ final class FeedViewController: UIViewController {
         tableView.tableHeaderView = segmentedControl
         segmentedControl?.addTarget(self, action: #selector(FeedViewController.indexChanged(_:)), for: .valueChanged)
 
+        refreshControl.attributedTitle = NSAttributedString(string: "Pull to refresh")
+        refreshControl.addTarget(self, action: #selector(refreshData), for: .valueChanged)
+        tableView.refreshControl = refreshControl
+    }
+
+    @objc
+    private func refreshData() {
+        loadMovies()
```

Comment:
You can directly call `loadMovies`
I mean you don't need to create `refreshData` method, you can add `@objc` to your `loadMovies` and pass it as a selector

Reply:
good idea